lsp-ui-doc.el

```diff
@@ -300,6 +300,10 @@ We don't extract the string that `lps-line' is already displaying."
          (lsp:marked-string-language contents))
     (lsp-ui-doc--extract-marked-string (lsp:marked-string-value contents)
                                        (lsp:marked-string-language contents)))
+   ;; The specification for MarkedString also includes raw strings of
+   ;; markdown, which is not reflected by `lsp-marked-string?'
+   ((eq (type-of contents) 'string)
```

Comment:
nit use: `stringp`

Reply:
Would you believe that I searched for this function and couldn't find it? I was looking for `string-p`. I'll make this change.